samples/tab-stage-view/nodejs/Readme.md

```diff
@@ -132,10 +132,18 @@ Also update `BaseUrl` according to your code runtime environment.
 ## Running the sample
 - In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
 
+**Install App:**
+
+![InstallApp](Images/Install.png)
+
 **Welcome message with feature explanation and Adaptive Card with actions:**
 
 ![Welcome Message](Images/welcomeAction.png)
 
+**Open the URL in tab stage view:**
```

Comment:
Please mention what will happen once you click on these buttons and add those screenshots as well.

Check all other places in Readme files.

Reply:
Fixed